macos/macos-api/build.gradle

```diff
@@ -15,17 +15,21 @@
  */
 
 plugins {
-    id 'java-library'
```

Comment:
@marcosholgado I need to navigate to MacOsActivity from my module, see `MacOsNav`. That interface has a dependency with Context and Intent, that's why we need to change this from java-library to android-library.